utils/environment.python.sh

```diff
@@ -0,0 +1,6 @@
+# FIXME: fasm should be installed into the running Python environment.
+export PYTHONPATH="${XRAY_DIR}:${XRAY_DIR}/third_party/fasm:$PYTHONPATH"
```

Comment:
This shouldn't be needed per https://github.com/SymbiFlow/prjxray/pull/1247/files#diff-b4ef698db8ca845e5845c4618278f29aR1